docs/hashes.toml

```diff
@@ -90,7 +90,7 @@ index = 1
 
 # Spec: ckb_staging
 [ckb_staging]
-spec_hash = "0x7ce187802e11bbdd074546f882d2608574ac20869baafcc1059a77d871e82733"
+spec_hash = "0x2d96bd61335be33418befa47f1c5ead53a0546b57a6d79719f0f999fa954dde4"
```

Comment:
The original value in v0.43.0:

https://github.com/nervosnetwork/ckb/blob/bbbe5a58f7024f1157fba75f26ab4e2fdfee1e38/docs/hashes.toml#L91-L93